README.md

```diff
@@ -6,10 +6,12 @@ MinIO uses [Sphinx](https://www.sphinx-doc.org/en/master/index.html) to generate
 
 ### Prerequisites
 
-- Any GNU/Linux Operating System.
+- Any GNU/Linux Operating System, or macOS 12.3 or later.
 - python 3.10.x and python-pip
+- python3.10-venv
 - sphinx 4.3.2
 - nodejs 14.5.0 or later
+- npm 16.19.1 or later
```

Comment:
This is also what our current Hugo build for kes-docs require so we might as well bump it.